README.md

```diff
@@ -5,11 +5,11 @@ Respond to short response questions in clear, concise sentences directly within
 
 ### Short Response Questions
 **1. What is recursion?**
-
+    Recursion is when a function is called within itself.
 **2. What type of problems are best solved using recursion?**
 
```

Comment:
Want you to think about this one! Another way to ask this question is: What are the similarities between the problems we've seen solved with recursion?